### DISCUSSION TITLE
🆘 Need Help: (A 1st Contact Pain): Configuration syntax for Variables

### DISCUSSION THREAD

**Discussion by iPoetDev** [super-linter/super-linter#4963] [Q&A] [answered]
In setting up Super Linter for the first time, I am running into a specific issue with referencing either the template linter configs or my own. I am referencing: [https://github.com/marketplace/actions/super-linter](https://github.com/marketplace/actions/super-linter)

I keep on getting this `(/action/lib/.automation/` before referencing the configs and it is frustrating in the tweaking/am getting into a knot. So I ask for a friend/self for help.

I have both the specific templates for each linter and my own linter rules.

when: 
1. "LINTER_RULES_PATH" = `/linters/` under `/.github/`
2. "LINTER_RULES_PATH" = `/`

```
70 2023-12-09 22:40:11 [INFO]   Gathering user validation information...
71 2023-12-09 22:40:11 [INFO]   - Only validating [new], or [edited] files in code base...
72 2023-12-09 22:40:12 [FATAL]     -> CSS_LINTER_RULES rules file (/action/lib/.automation/./../.stylelintrc.json) doesn't exist. Terminating...
```

This is my repo:  https://github.com/iPoetDev/AlpineTicTacToe/
This is my Pull Request: https://github.com/iPoetDev/AlpineTicTacToe/pull/4
This is my actions https://github.com/iPoetDev/AlpineTicTacToe/actions

- Structure: `https://github.com/iPoetDev/AlpineTicTacToe/tree/main/.github`
- I have set a `/.github/super-linter.env` for local and remote CI.
- and (err, sorry) I also have the variables in the workflow file: `/.github/workflow/super-linter.yml`

This is my first time with CI/workflows (1st Contact) so I am using this as a learning curve.

**Comment by zkoppert** [comment, marked as answer]
It looks like you have your linter configuration files split across the `.github/linters/` directory and the `/` directory. I would consolidate those into the `.github/linters/` location. It also appears you are running the super linter twice: once at `.github/workflows/super-linter.yml` and again at `.github/workflows/ci.yml`. I would consolidate to use just one of those as a start. 

Also, I notice that you have many different configurations and customizations. I would see if you can get the super-linter working in its base configuration and then try modifying things one by one so you can tell what configuration change is causing issue.

**Reply by PILBABA** [reply]
Good
